charts/layer7-operator/crds/gateway-crd.yaml

```diff
@@ -2048,39 +1689,51 @@ spec:
                                   certificate contents
                                 type: string
                               certificate:
-                                description: certificate provides certificate contents
+                                description: certificate provides certificate contents.
                                 type: string
                               destinationCACertificate:
                                 description: destinationCACertificate provides the
-                                  contents of the ca certificate of the final destination.  When
-                                  using reencrypt termination this file should be
-                                  provided in order to have routers use it for health
-                                  checks on the secure connection. If this field is
-                                  not specified, the router may provide its own destination
-                                  CA and perform hostname validation using the short
-                                  service name (service.namespace.svc), which allows
-                                  infrastructure generated certificates to automatically
-                                  verify.
+                                  contents of the ca certificate of the
                                 type: string
+                              externalCertificate:
+                                description: externalCertificate provides certificate
+                                  contents as a secret reference.
+                                properties:
+                                  name:
+                                    description: |-
+                                      name of the referent.
+                                      More info: https://kubernetes.
+                                    type: string
+                                type: object
+                                x-kubernetes-map-type: atomic
                               insecureEdgeTerminationPolicy:
-                                description: "insecureEdgeTerminationPolicy indicates
-                                  the desired behavior for insecure connections to
-                                  a route. While each router may make its own decisions
-                                  on which ports to expose, this is normally port
-                                  80. \n * Allow - traffic is sent to the server on
-                                  the insecure port (default) * Disable - no traffic
-                                  is allowed on the insecure port. * Redirect - clients
-                                  are redirected to the secure port."
+                                description: insecureEdgeTerminationPolicy indicates
+                                  the desired behavior for insecure c
```

Comment:
incomplete sentence, is this expected ?

Reply:
Yes - set the maxDescLength to 75. If this isn't done, the CRDs become too large to install.

```
.PHONY: manifests
manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
#$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:generateEmbeddedObjectMeta=true,maxDescLen=75 webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Reply:
The /docs are not limited by this and contain the full description